### PR TITLE
sane_backends: Fixed 1.0.25 gcc2 build , added PNG

### LINKED ISSUE #4143: sane_backends: new release 1.0.28
https://gitlab.com/sane-project/backends/-/releases

### PR DESCRIPTION
**Pull request by kenmays**
Tested on hrev53452.
Fixed download URL issue.
Added PNG.
Cleanup.
Closes #4143

### CODE REVIEW

File: media-gfx/sane_backends/sane_backends-1.0.25.recipe [hunk 1]
```diff
@@ -81,21 +82,18 @@ BUILD_PREREQUIRES="
...
 		--prefix=$prefix \
 		--bindir=$binDir \
 		--sbindir=$binDir \
 		--libdir=$libDir \
 		--includedir=$includeDir \
 		--datadir=$dataDir \
-		--with-docdir=$docDir \
+		--docdir=$docDir \
 		--enable-pthread \
+		--disable-ipv6 \
```
Comment (waddlesplash) [inline media-gfx/sane_backends/sane_backends-1.0.25.recipe]: This is definitely wrong, we don't support it really yet.

File: media-gfx/sane_backends/sane_backends-1.0.25.recipe [hunk 2]
```diff
@@ -81,21 +82,18 @@ BUILD_PREREQUIRES="
...
-	automake
-
 	runConfigure configure \
 		--prefix=$prefix \
 		--bindir=$binDir \
 		--sbindir=$binDir \
 		--libdir=$libDir \
 		--includedir=$includeDir \
 		--datadir=$dataDir \
-		--with-docdir=$docDir \
```
Comment (waddlesplash) [inline media-gfx/sane_backends/sane_backends-1.0.25.recipe]: None of the above lines should be needed.

File: media-gfx/sane_backends/sane_backends-1.0.25.recipe (hunk 1 above)
Comment (korli) [reply media-gfx/sane_backends/sane_backends-1.0.25.recipe]: what do you mean with " we don't support it really yet"?

File: media-gfx/sane_backends/sane_backends-1.0.25.recipe [hunk 3]
```diff
@@ -18,17 +18,16 @@ documentation.
...
 CHECKSUM_SHA256="a4d7ba8d62b2dea702ce76be85699940992daf3f44823ddc128812da33dc6e2c"
 SOURCE_DIR="sane-backends-$portVersion"
 PATCHES="
 	sane_backends-$portVersion.patchset
 	sane_disable_libusb_context.patchset
 	"
 
-ARCHITECTURES="x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+ARCHITECTURES="x86_gcc2"
```
Comment (korli) [inline media-gfx/sane_backends/sane_backends-1.0.25.recipe]: Please restore ARCHITECTURES and SECONDARY_ARCHITECTURES.

Comment (kenmays) [reply media-gfx/sane_backends/sane_backends-1.0.25.recipe]: fixed

File: media-gfx/sane_backends/sane_backends-1.0.25.recipe (hunk 2 above)
Comment (kenmays) [reply media-gfx/sane_backends/sane_backends-1.0.25.recipe]: fixed

**Pull request closed without merging**